backend/server/routers/model.py

```diff
@@ -81,8 +81,6 @@ class CoursesUnlockedWhenTaken (BaseModel):
     direct_unlock: list
     indirect_unlock: list
 
```

Comment:
at the top of this file, add
```
#pylint: disable=missing-class-docstring
```
since we dont care about this warning for models.